src/components/pill/container/index.js

```diff
@@ -24,12 +24,12 @@ const Container = styled(Flex).attrs(
     },
     height: getPillHeight(height, tiny),
     width: getPillWidth(width, tiny),
-    cursor: isDisabled ? "not-allowed" : "pointer",
     justifyContent: "center",
     alignItems: "center",
   })
 )`
   ${getBackground};
+  cursor: ${({ isDisabled }) => (isDisabled ? "not-allowed" : "pointer")};
```

Comment:
There is `cursor` mixin already available [here](https://github.com/netdata/netdata-ui/blob/master/src/mixins/cursor.js). Let's utilize it.